tableauserverclient/server/request_factory.py

```diff
@@ -469,6 +469,90 @@ def update_req(self, site_item):
+            site_element.attrib['userQuota'] = str(site_item.user_quota).lower()
+        if site_item.tier_creator_capacity is not None:
+            site_element.attrib['tierCreatorCapacity'] = str(site_item.tier_creator_capacity).lower()
+        if site_item.tier_explorer_capacity is not None:
+            site_element.attrib['tierExplorerCapacity'] = str(site_item.tier_explorer_capacity).lower()
+        if site_item.tier_viewer_capacity is not None:
+            site_element.attrib['tierViewerCapacity'] = str(site_item.tier_viewer_capacity).lower()
+        if site_item.data_alerts_enabled is not None:
+            site_element.attrib['dataAlertsEnabled'] = str(site_item.data_alerts_enabled).lower()
+        if site_item.commenting_mentions_enabled is not None:
+            site_element.attrib['commentingMentionsEnabled'] = str(site_item.commenting_mentions_enabled).lower()
+        if site_item.catalog_obfuscation_enabled is not None:
+            site_element.attrib['catalogObfuscationEnabled'] = str(site_item.catalog_obfuscation_enabled).lower()
+        if site_item.flow_auto_save_enabled is not None:
+            site_element.attrib['flowAutoSaveEnabled'] = str(site_item.flow_auto_save_enabled).lower()
+        if site_item.web_extraction_enabled is not None:
+            site_element.attrib['webExtractionEnabled'] = str(site_item.web_extraction_enabled).lower()
+        if site_item.run_now_enabled is not None:
+            site_element.attrib['runNowEnabled'] = str(site_item.run_now_enabled).lower()
+        if site_item.metrics_content_type_enabled is not None:
+            site_element.attrib['metricsContentTypeEnabled'] = str(site_item.metrics_content_type_enabled).lower()
+        if site_item.notify_site_admins_on_throttle is not None:
+            site_element.attrib['notifySiteAdminsOnThrottle'] = str(site_item.notify_site_admins_on_throttle).lower()
+        if site_item.authoring_enabled is not None:
+            site_element.attrib['authoringEnabled'] = str(site_item.authoring_enabled).lower()
+        if site_item.custom_subscription_email_enabled is not None:
+            site_element.attrib['customSubscriptionEmailEnabled'] = \
+                str(site_item.custom_subscription_email_enabled).lower()
+        if site_item.custom_subscription_email is not None:
+            site_element.attrib['customSubscriptionEmail'] = str(site_item.custom_subscription_email).lower()
+        if site_item.custom_subscription_footer_enabled is not None:
+            site_element.attrib['customSubscriptionFooterEnabled'] =\
+                str(site_item.custom_subscription_footer_enabled).lower()
+        if site_item.custom_subscription_footer is not None:
+            site_element.attrib['customSubscriptionFooter'] = str(site_item.custom_subscription_footer).lower()
+        if site_item.ask_data_mode is not None:
+            site_element.attrib['askDataMode'] = str(site_item.ask_data_mode).lower()
```

Comment:
askDataMode is not a boolean, and I believe it is case-sensitive. The `.lower()` should be removed.
Same applies in create_req().